<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDVMainWindow</class>
 <widget class="QMainWindow" name="MDVMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/main-icon.png</normaloff>:/icons/main-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="MediaSettingsGroupBox" name="mediaSettinsGroupBox">
       <property name="title">
        <string>Media Settings</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="Screens">
       <property name="title">
        <string>Screens</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyTabWidget" name="screensTabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionRenameScreen"/>
    <addaction name="separator"/>
    <addaction name="actionExtensions"/>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="title">
     <string>&amp;Service</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionLog"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Project"/>
   <addaction name="menuService"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/qt.png</normaloff>:/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/question-frame.png</normaloff>:/icons/question-frame.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRenameScreen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Rename Screen</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder-open-document.png</normaloff>:/icons/folder-open-document.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document--plus.png</normaloff>:/icons/document--plus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/door-open.png</normaloff>:/icons/door-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/disk--pencil.png</normaloff>:/icons/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Save Project As</string>
   </property>
  </action>
  <action name="actionExtensions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/plug.png</normaloff>:/icons/plug.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Extensions</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exclamation-button.png</normaloff>:/icons/exclamation-button.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/wrench.png</normaloff>:/icons/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTabWidget</class>
   <extends>QTabWidget</extends>
   <header>mytabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MediaSettingsGroupBox</class>
   <extends>QGroupBox</extends>
   <header>mediasettingsgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MDVMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
